python/ray/util/accelerators/__init__.py

```diff
@@ -24,6 +25,7 @@
     "NVIDIA_TESLA_A10G",
     "INTEL_MAX_1550",
     "INTEL_MAX_1100",
+    "INTEL_GAUDI",
```

Comment:
Based on your `get_current_node_accelerator_type` doc string, there is also Gaudi2.

Also is it GAUDI or Gaudi?

Reply:
GAUDI